packages/dara-core/js/types/core.tsx

```diff
@@ -188,7 +188,7 @@ export interface BaseComponentProps {
  */
 export interface BaseStylingProps {
     /** How to align component, either itself or children within */
-    align?: 'start' | 'end' | 'left' | 'right' | 'center' | 'justify' | 'match-parent';
+    align?: string;
```

Comment:
Isn't `align` also a layout component specific prop? Why does it belong to the base styling props? Doesn't seem to be handled by `useComponentStyles` so should probably be moved as well

Reply:
Other components may also have `align` for example you can apply it to `Text` which is not a layout component

Reply:
Shouldn't Text define align then, as it's a different align (`text-align`) than the flex align (`align-items`) I believe?

Reply:
On the text component it does result in text-align. I didn't remove it from here because other components can define align differently and it was not unique to layout. But we can have it individually defined if not in layout components as well.

Reply:
Right. Perhaps that's a future improvement, we should eventually disentangle these, they really shouldn't have the same name.

Reply:
Yes we can think about it and see what we want to do :) 
I do see an argument of it being called the same for simplicity since it is the same concept and one thing to understand and remember for someone not familiar with css.  But also adds a bit of magic of it being translated to different things underneath depending on the component it is used on.

Reply:
Sure, but at the very least the prop should be defined on Text itself explaining what it maps to rather than on the very base components, as some components might not implement align at all.